### PR TITLE
feat: new alerts for ETH and LINK balance on cross-chain bridge

### PR DESCRIPTION
**Pull request by Ivan-Feofanov**
New alerts:
- Runs every 7200 blocks (once a day)
- Checks that ETH balance of CrossChainController is higher or equal 0.5 ETH
- Checks that LINK balance of CrossChainController is higher or equal 5 LINK

- [x] Write code
- [x] Write tests
- [x] Add new alerts to docs

### CODE REVIEW

File: l2-bridge-ethereum/src/watchers/bridge-watcher/handlers.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        alertId: "BRIDGE-ETH-BALANCE-LOW",
+        severity: FindingSeverity.Medium,
+        type: FindingType.Info,
+        metadata: { ethBalance: ethBalance.toString() },
+      }),
+    );
+  }
+
+  // Check LINK balance of the bridge
+  const linkContract = new ethers.Contract(
```
Comment (BATMAH69) [inline l2-bridge-ethereum/src/watchers/bridge-watcher/handlers.ts:69]: you can get 2 balance with promise all and you need to handle network error for queries

**Review by BATMAH69 — CHANGES_REQUESTED** [review]

**Review by BATMAH69 — APPROVED** [review]

File: l2-bridge-ethereum/src/constants.ts [hunk 2]
```diff
@@ -23,6 +23,13 @@ export interface BridgeProxyInfo {
   proxyAdminAddress: string | null;
 }
 
+export const LINK_TOKEN_ADDRESS = "0x514910771AF9Ca656af840dff83E8264EcF986CA";
```
Comment (tamtamchik) [inline l2-bridge-ethereum/src/constants.ts:26]:
```suggestion
// https://docs.chain.link/resources/link-token-contracts#ethereum-mainnet
export const LINK_TOKEN_ADDRESS = "0x514910771AF9Ca656af840dff83E8264EcF986CA";
```

File: l2-bridge-ethereum/src/constants.ts [hunk 3]
```diff
@@ -23,6 +23,13 @@ export interface BridgeProxyInfo {
...
+export const CROSS_CHAIN_CONTROLLER_PROXY_ADDRESS =
```
Comment (tamtamchik) [inline l2-bridge-ethereum/src/constants.ts:27]:
```suggestion
// https://docs.lido.fi/deployed-contracts/#adi-governance-forwarding
export const CROSS_CHAIN_CONTROLLER_PROXY_ADDRESS =
```

File: l2-bridge-ethereum/src/watchers/bridge-watcher/handlers.ts [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+            metadata: { args: String(event.args) },
+          }),
+        );
+      });
+    }
+  });
+}
+
+export async function handleBridgeBalance(event: BlockEvent) {
+  if (event.block.number % 7200 !== 0) {
```
Comment (tamtamchik) [inline l2-bridge-ethereum/src/watchers/bridge-watcher/handlers.ts:44]: Is it possible that forta will bypass some block?

**Review by tamtamchik — DISMISSED** [review]
LGTM 👍 

Although, I have added some suggestions and a question.

File: l2-bridge-ethereum/src/watchers/bridge-watcher/handlers.ts (hunk 4 above)
Comment (Ivan-Feofanov) [reply l2-bridge-ethereum/src/watchers/bridge-watcher/handlers.ts:44]: Hardly, but we live in the world where everything is possible

Comment (tamtamchik) [reply l2-bridge-ethereum/src/watchers/bridge-watcher/handlers.ts:44]: That was the exact reason for my question :) is it costly for us to check every block?

Comment (tamtamchik) [reply l2-bridge-ethereum/src/watchers/bridge-watcher/handlers.ts:44]: Agreed to check once an hour.

**Review by tamtamchik — APPROVED** [review]

**Pull request merged**